R/target_market_share.R

```diff
@@ -419,6 +285,37 @@ calculate_ald_benchmark <- function(ald, region_isos, by_company) {
   out
 }
 
+add_name_ald_if_by_company <- function(data, by_company = FALSE) {
+  if (by_company) {
+    data <- c(data, "name_ald")
+  }
+
+  data
+}
```

Comment:
nit: is `data` a vector here instead of a data frame as it seems to be most other places in this script? hard to tell without any validation

Reply:
i.e. if it is a vector, I'd expect this to work properly `data <- c(data, "name_ald")`... but if it's a data frame, I'm not sure what would happen
```r
data <- data.frame(x = 1:2); c(data, "test")
#> $x
#> [1] 1 2
#> 
#> [[2]]
#> [1] "test"
```

Reply:
It is a vector. I just default to using `data` as the first argument of most functions, but you are right, i will disambiguate.